server/src/main/java/org/opensearch/cluster/InternalClusterInfoService.java

```diff
@@ -569,8 +569,8 @@ public void onFailure(Exception e) {
 
         @Override
         public void onRejection(Exception e) {
-            final boolean shutDown = e instanceof OpenSearchRejectedExecutionException
-                && ((OpenSearchRejectedExecutionException) e).isExecutorShutdown();
+            final boolean shutDown = e instanceof OpenSearchRejectedExecutionException openSearchRejectedExecutionException
+                && openSearchRejectedExecutionException.isExecutorShutdown();
```

Comment:
nit: given openSearchRejectedExecutionException is very shortlived, let's call it something small - maybe just `osre` suffices.

You may choose to not resolve this as doesn't matter much honestly.